requirements.yml

```diff
@@ -3,10 +3,11 @@ channels:
     - conda-forge
     - defaults
 dependencies:
-    - python==3.8
+    - python==3.9
```

Comment:
i am working with @JoaoRodrigues to understand what in FCCpy is incompatible with python 3.8. If there is no special reason we should keep it `3.8` compatible.